src/strategies/stakedao-governance-update/README.md

```diff
@@ -0,0 +1,18 @@
+# sd-boost
```

Comment:
`sd-boost`? 🤔

Reply:
Forget too change it from the previous snapshot for StakeDAO, I've update it on the new commit 😅